From 253346878c9127afbe60e69f0afa3e64632a20c7 Mon Sep 17 00:00:00 2001
From: David Robillard <d@drobilla.net>
Date: Thu, 11 Mar 2021 13:33:42 -0500
Subject: [PATCH] Remove subclass cycle

It's unfortunate to diverge from the canonical document, but most of these
third-party schemas already do anyway.  This may be technically fine according
to rdfs (I'm not sure), but it's better for the LV2 universe as a whole to
consistently eliminate such things.
---
 schemas.lv2/dct.ttl | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/schemas.lv2/dct.ttl b/schemas.lv2/dct.ttl
index 5cdd5047..d8fd1962 100644
--- a/schemas.lv2/dct.ttl
+++ b/schemas.lv2/dct.ttl
@@ -22,8 +22,7 @@ dcterms:AgentClass
 	dcterms:description "Examples of Agent Class include groups seen as classes, such as students, women, charities, lecturers."@en-us ;
 	rdfs:comment "A group of agents."@en-us ;
 	rdfs:isDefinedBy dcterms: ;
-	rdfs:label "Agent Class"@en-us ;
-	rdfs:subClassOf dcterms:AgentClass .
+	rdfs:label "Agent Class"@en-us .
 
 dcterms:LicenseDocument
 	a rdfs:Class ;
